zmsadmin/js/page/availabilityDay/form/validate.js

```
@@ -1,36 +1,169 @@
+        errorList.push({
+            type: 'endTimeFormat',
+            message: 'Die Endzeit muss im Format "HH:mm:ss" oder "HH:mm" vorliegen.'
+        });
+    }
+
+    return errorList;
+}
+
+// Helper function to check if a timestamp is valid
+function isValidTimestamp(timestamp) {
+    return !isNaN(timestamp) && moment.unix(timestamp).isValid();
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)